Servers/driver/autoDriver.driver.ts

```
@@ -408,11 +408,11 @@ export async function insertMockData() {
   } = insertQuery["projectMembers"];
   let projectMembers;
   if (projectMembersMockData.length !== 0) {
-    const values = projectMembersMockData(users![0].id, users![1].id, projects![0].id, projects![1].id).map((d) =>
+    const values = projectMembersMockData(users![0].id!, users![1].id!, projects![0].id!, projects![1].id!).map((d) =>
       projectMembersGenerateValuesString(d as any)
     );
     insertString += values.join(",") + ";";
-    projectMembers = await insertData(insertString as string);
+    projectMembers = (await insertData(insertString as string) as [{}[], number])[0];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)